tor/cmd_onion.go

```diff
@@ -254,3 +257,38 @@ func (c *Controller) AddOnion(cfg AddOnionConfig) (*OnionAddr, error) {
 		Port:         cfg.VirtualPort,
 	}, nil
 }
+
+// DelOnion tells the server to remove an Onion ("Hidden") Service, that was
+// previously created via an "ADD_ONION" command. It is only possible to remove
+// Onion Services that were created on the same control connection as the
+// "DEL_ONION" command, and those that belong to no control connection in
+// particular (The "Detach" flag was specified at creation).
```

Comment:
nit: clarify this "and those that belong to no control connection in particular if the "Detach" flag ..."

Reply:
Updated!